### PR TITLE
LPS-162655 [PRM] Create Objects MDF Claim

### PR DESCRIPTION
**Pull request by DiogoNo**

### CODE REVIEW

**Comment by DiogoNo** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/object-relationships/mdf-request-to-mdf-claim.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/object-relationships/mdf-request-to-mdf-claim.json:1]: `mdf-request-to-mdf-claims.json`

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/object-relationships/mdf-claim-to-mdf-claim-documents.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+	"deletionType": "cascade",
+	"label": {
+		"en-US": "MDF Claim to MDF Claim Document"
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/object-relationships/mdf-claim-to-mdf-claim-documents.json:4]:
```suggestion
		"en-US": "MDF Claim to MDF Claim Documents"
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/object-definitions/mdf-claim-document.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+		{
+			"DBType": "String",
+			"businessType": "Picklist",
+			"indexed": true,
+			"indexedAsKeyword": false,
+			"label": {
+				"en_US": "Type"
+			},
+			"listTypeDefinitionId": "[$LIST_TYPE_DEFINITION_ID:MDF Document Types$]",
+			"name": "typeDocuments",
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/object-definitions/mdf-claim-document.json:28]:
```suggestion
			"name": "type",
```

**Review by jairmedeiros — CHANGES_REQUESTED** [review]

**Comment by jairmedeiros** [comment]
socorro

**Review by jairmedeiros — APPROVED** [review]

**Comment by jairmedeiros** [comment]
ci:forward

**Pull request closed without merging**